text/0085-api-keys.md

```diff
@@ -41,15 +41,16 @@ At the first launch of Meilisearch with a master key, Meilisearch automatically
 
 If the master key is removed at Meilisearch launch, the previously generated API keys no longer secure the Meilisearch instance.
 
-If Meilisearch is launched with the `production` value for the `MEILI_ENV` environment variable or the `--env` CLI option, a master key is mandatory. If the master key is omitted in that particular case, Meilisearch launch is aborted and displays an error:
+If Meilisearch is launched with the `production` value for the `MEILI_ENV` environment variable or the `--env` CLI option, a master key of at least 16 characters is mandatory. If the master key is omitted in that particular case, or is too short, Meilisearch launch is aborted and displays an error:
 
-`Error: In production mode, the environment variable MEILI_MASTER_KEY is mandatory`
+- `Error: In production mode, the environment variable MEILI_MASTER_KEY is mandatory` if the key is omitted.
```

Comment:
```suggestion
- `Error: In production mode, you must provide a master key to secure your instance. It can be specified via the MEILI_MASTER_KEY environment variable or the --master-key launch option.`
```

Hey @dureuill, we had a [discussion](https://github.com/meilisearch/product/discussions/565) dedicated to that particular error. I'm suggesting the one from @dichotommy here!

Source: https://github.com/meilisearch/product/discussions/565#discussioncomment-4266375

Reply:
Thank you @gmourier! I agree with the change. Unfortunately, the implementation PR for this has already landed. I will have to open another one.

Reply:
I noticed that tamo already fixed that on main. Thanks, @irevoire ❤️

Reply:
Updated in the spec!